web_classification/flows/experiment/flow.dag.yaml

```diff
@@ -39,7 +39,7 @@ nodes:
     text_content: ${summarize_text_content.output}
     examples: ${prepare_examples.output}
   provider: AzureOpenAI
-  connection: aoai
+  connection: aoai_conn2
```

Comment:
is there a better name for "aoai_conn2", something short that surfaces the distinction between the AOAI instances?

Reply:
This is just what the connection that @kottofy made was called and I didn't know how she did it until later so just changed the name rather than update to a new connection

Reply:
gotcha